ds_judgements_public_ui/sass/includes/_service_introduction.scss

```diff
@@ -1,16 +1,22 @@
 .service-introduction {
   background-color: $color-almost-black;
   flex-grow: 1;
+  width: 100%;
+  margin-top: $space-10;
 
   &__header {
     font-size: $typography-2xl-text-size;
-    line-height: 3.8rem;
-    display: inline-block;
+    line-height: $typography-md-line-height;
+
     width: 100%;
     font-family: $font-roboto;
     font-weight: normal;
     color: $color-white;
-    margin: 0;
+    margin: 0 $space-1;
+    @media only screen and (max-width: $grid-breakpoint-medium) {
+      display: inline-block;
+      float: right;
```

Comment:
We should avoid using floats as much as possible - we shouldn't need it in this situation, especially if we style with flex box as mentioned in the previous comment